circle.yml

```diff
@@ -25,16 +25,20 @@ dependencies:
     - DEBIAN_FRONTEND=noninteractive sudo apt-get autoremove -y --purge
         libblas-dev
         libbz2-dev
+        libexpat1-dev
```

Comment:
> I don't know whether the the :python requirement currently checks that the Python header/libraries are installed. That'd be needed for option 2.

I quickly glanced at it; looks like it doesn't.

I'm approving the changes now; we can postpone this discussion until after 1.1.12 is merged.